app/src/main/AndroidManifest.xml

```diff
@@ -2,8 +2,12 @@
 <manifest package="fmuntenescu.playground"
     xmlns:android="http://schemas.android.com/apk/res/android">
 
+    <!--Uncomment the read and write storage rights when using method tracing-->
+    <!--<uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>-->
+    <!--<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE"/>-->
```

Comment:
keep it simply, make your min sdk 4.4. then you dont need this anymore as long as you stick to getExternalFilesDir())
https://developer.android.com/guide/topics/manifest/uses-permission-element.html